src/apps/loan-list/materials/stackable-material/stackable-material.tsx

```diff
@@ -44,7 +44,7 @@ const StackableMaterial: FC<StackableMaterialProps & MaterialProps> = ({
         "list-reservation--stacked": additionalMaterials > 0
       })}
       role="button"
-      onClick={() => openLoanDetailsModalHandler()}
+      onClick={() => openDueDateModal && dueDate && openDueDateModal(dueDate)}
       onKeyUp={(e) => {
         if (e.key === "Enter" || e.key === "Space") {
           openLoanDetailsModalHandler();
```

Comment:
We should have the same behavior here, right?

Reply:
Good eyes, sharp mind! :)) Thx